backend/api/appsettings.json

```diff
@@ -1,23 +1,24 @@
 {
+  "BaseUrl": "/api",
+  "HealthChecks": {
+    "Port": "8080"
+  },
   "HealthChecksUI": {
-    "HealthChecks": [
-      {
+    "HealthChecks": [{
         "Name": "liveliness",
-        "Uri": "http://localhost:8080/api/live"
+        "Uri": "http://localhost:8080/api/health/live"
```

Comment:
Changed route to associate it with related endpoints.  @devinleighsmith Will this break OpenShift?

Reply:
Yes, it will. No problem, we just need to update the deploy config once this PR lands